resources/images/controller/images.go

```diff
@@ -330,6 +331,30 @@ func (s *SoftwareImagesController) handleImage(
 	return tmpfile, meta, http.StatusOK, nil
 }
 
+func getArtefactInfo(info *metadata.Info) *images.ArtifactInfo {
```

Comment:
tomato, tomahto

You use artifact in other places, why make this func a special snowflake?